tests/core/test_serializable.py

```diff
@@ -242,12 +242,14 @@ def test_container_deserialize_bad_values(value, sedes):
 def test_empty_fields_container_deserialize():
     with pytest.raises(DeserializationError):
         SSZType5.deserialize(SSZType6.serialize(_type_6))
+    with pytest.raises(DeserializationError):
         SSZType6.deserialize(SSZType5.serialize(_type_5))
 
```

Comment:
Would like to see a case that explicitly has a 2 fields container failed on parsing a 3 elements bytes. https://github.com/ethereum/eth2.0-specs/issues/479#issuecomment-456046599

Reply:
It's here: https://github.com/ethereum/py-ssz/blob/2bba715f9f9857b360045804bd9e968e4b97d4d5/tests/core/test_serializable.py#L250-L251 (with `uint32` instead of `bytes` though), or do you mean something else?

Reply:
Oh, yes, that should work.
I tested this line before your change, and it didn't raise an exception 😓. Good catch on line 252.